web-console/src/views/ingestion-view/ingestion-view.tsx

```diff
@@ -244,6 +247,8 @@ ORDER BY "rank" DESC, "created_time" DESC`;
       hiddenSupervisorColumns: new LocalStorageBackedVisibility(
         LocalStorageKeys.SUPERVISOR_TABLE_COLUMN_SELECTION,
       ),
+
+      preferOverlord: true,
```

Comment:
this would change the default behavior, so we should make sure we are understand the implications for the user and call this out in the release notes. @gianm @vogievetsky any opinion on whether we should keep the existing default or not?

Reply:
I'm just catching up here, but why does using the Overlord API cause less stress on the metadata store than using SQL? Under the hood, I believe that the SQL sys.tasks table would be making Overlord API calls anyway.

So I guess the calls are different. But: how are they different?

Ideally I'd like to take that knowledge and use it to make the sys.tasks implementation more efficient.

Also, does using the Overlord API directly have any implications for functionality? (Is it reduced in some way?)

Reply:
@gianm , you are right. I think the confusion was between the APIs "/druid/indexer/v1/task" and "/druid/indexer/v1/tasks". First one for posting new tasks(thru ingestion specs) and the second one that is being used for fetching the task list. Second one is going to the backend metadata store. Please ignore this PR. 

But the underlying problem is that the data model for druid_tasks table has task status embedded inside "status_payload" and we need to fetch all the tasks to UI in order to filter them as running, success and failed. At our current scale we are fetching a lot of data from RDS and this is causing network throughput increase and affecting performance of the system. Ideal way would be to remodel the table to have status as a field which can queried and filter at metadata store itself as explained https://github.com/apache/druid/issues/12318. 

Hope this helps.